addons/mail_group/tests/test_mail_group_moderation.py

```diff
@@ -137,13 +121,6 @@ def setUpClass(cls):
         super(TestModeration, cls).setUpClass()
 
         # Test group: members, moderation
-        cls.test_group_2 = cls.env['mail.group'].create({
```

Comment:
Why this change ? At merge I tried to create as few data as possible to keep tests contained ... otherwise it quickly becomes a mess.

Reply:
I moved it to the setUp method (not anymore in the common file, so it's created just for those tests)